Clinical-Genomics/scout
# Fix #1792 - assume GATK caller.

Hm, I thought I made a PR with this? Lost in translation somewhere? Anyway, this is a fallback fix for #1792, until MIP is back on track on the issue again, but also a potentially welcome addition for other users running GATK only on the SNV side.

In principle it is ready for review, but if you wait a test or two may appear. 😊 

**How to test**:
1. load SNV variants without INFO.FOUND_IN or INFO.set information about caller set and note that GATK pass/filer status is evident on the variants.

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
